src/api/import.js

```diff
@@ -161,14 +161,25 @@ export default async (providedOptions = {}) => {
     d('deleting old dependencies forcefully');
     await fs.remove(path.resolve(dir, 'node_modules/.bin/electron'));
     await fs.remove(path.resolve(dir, 'node_modules/.bin/electron.cmd'));
-    await fs.remove(path.resolve(dir, 'node_modules', electronName));
+
+    if (electronName) {
+      await fs.remove(path.resolve(dir, 'node_modules', electronName));
+    }
 
     d('installing dependencies');
     await installDepList(dir, deps);
+
     d('installing devDependencies');
     await installDepList(dir, devDeps, true);
-    d('installing electron-prebuilt-compile');
-    await installDepList(dir, [`electron-prebuilt-compile@${electronVersion || 'latest'}`], false, true);
+
+    d('installing exactDevDependencies');
+    const ePrebuiltCompilePos = exactDevDeps.indexOf('electron-prebuilt-compile');
```

Comment:
This can look a bit nicer code-wise using some ES6 *magics*

```js
await installDepList(dir, exactDevDeps.map((dep) => {
  if (dep ==='electron-prebuild-compile') {
    return `electron-prebuilt-compile@${electronVersion || 'latest'}`;
  }
  return dep;
}), true, true);
```

Just means we aren't mutating the array here as well (`.splice` mutates)